frontend/components/SetupPage/Create/SetupEoaFunding.tsx

```diff
@@ -135,135 +126,128 @@ export const SetupEoaFundingForChain = ({
         The app needs these funds to create your account on-chain.
       </Paragraph>
 
-      <CardSection
-        padding="12px 24px"
-        bordertop="true"
-        borderbottom={isFunded ? 'true' : 'false'}
-      >
+      <CardSection padding="12px 24px" bordertop="true" className="mt-12">
         <Text className={isFunded ? '' : 'loading-ellipses'}>
           Status: {statusMessage(isFunded)}
         </Text>
       </CardSection>
-      {!isFunded && <SetupEoaFundingWaiting chainName={chainName} />}
+      {!isFunded && (
+        <SetupEoaFundingWaiting chainName={chainName} currency={currency} />
+      )}
     </CardFlex>
   );
 };
 
+type EoaFundingMapParams = {
+  provider: ethers.providers.JsonRpcProvider;
+  chainConfig: ChainConfig;
+  requiredEth: number;
+};
+
 // TODO: chain independent
-const eoaFundingMap = {
-  //   [MiddlewareChain.OPTIMISM]: {
-  //     provider: OPTIMISM_PROVIDER,
-  //     chainConfig: CHAIN_CONFIG.OPTIMISM,
-  //     requiredEth:
-  //       MIN_ETH_BALANCE_THRESHOLDS[MiddlewareChain.OPTIMISM].safeCreation,
+// use SERVICE_TEMPLATES[].configurations instead?
```

Comment:
I'll probably rewrite this to use SERVICE_TEMPLATES as done in SetupCreateSafe component in next prs, but lmk your thoughts anyway